source/class/qx/ui/treevirtual/SimpleTreeDataCellRenderer.js

```diff
@@ -408,7 +408,8 @@ qx.Class.define("qx.ui.treevirtual.SimpleTreeDataCellRenderer", {
       var node = cellInfo.value;
 
       // Add the node's icon
-      var imageUrl = node.bSelected ? node.iconSelected : node.icon;
+      var imageUrl =
+        bSelected && node.iconSelected ? node.iconSelected : node.icon;
```

Comment:
This looks wrong. `bSelected` appears to be an undefined variable (unless it's global, which seems unlikely). Shouldn't that be `node.bSelected`?

Reply:
Thank you, my bad. Looks like copied wrongly from one project to another to make commit.